apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Using specific templates in PDPs and PLPs'
+sidebar_label: 'Multiple page templates'
+---
+
+<header>
+
+# Multiple page templates
+
+</header>
+
+Multiple Page Template means that PDPs and PLPs, previously defined as singleton in the CMS, can now assume different page templates based on some rules. This approach is almost similar to the PDP and PLP, but they will be explained separately since each page type has particularities.
```

Comment:
Does the person need to add these content types in their store code, or do they already come in the package (maybe just need to update it)?

Reply:
Just to understand if the doc should have, as an objective key, a tutorial on adding these content types, using them in the Headless CMS, or both. 🔎